### DISCUSSION TITLE
Feature request: header in popup should also function as a slider

### DISCUSSION THREAD

**Discussion by jorisvanr** [Clooos/Bubble-Card#284] [Feature requests]
Hi! I'm experimenting with the popup card and I think it is exactly what I was missing. I currently show a slider when my lights are on and hide it again when the lights turn off. This would however be redundant if the header (see attached image) would also function as a slider itself. 

I might be missing something since this is how it works when the button type is set to a slider. Having a similar option in the popup would be huge so I can get rid of another feature 🎉

ps. I'm also noticing font size issues in these screenshots but I'm not sure if that is a conflict on my end. Did not look into that yet. 

<img width="523" alt="image" src="https://github.com/Clooos/Bubble-Card/assets/16778969/68f579e2-8532-465b-8200-2a19001d659e">
<img width="602" alt="image" src="https://github.com/Clooos/Bubble-Card/assets/16778969/02b8b53b-bdea-4d08-a589-2739502f867d">

**Comment by TurboNimitz** [comment]
I think I'm on the same side here. A group of lights controlled by a slider card but when adding popup to it the slider for the group doesn't work anymore. Could be interperenting your feature request wrong here or is it the same?
